pkg/metrics/constants.go

```diff
@@ -37,6 +37,9 @@ const (
 	ExpirationReason    = "expiration"
 	EmptinessReason     = "emptiness"
 	DriftReason         = "drift"
+
+	GracefulDisruptionClass = "graceful" // graceful disruption always respects blocking pod PDBs and the do-not-disrupt annotation
+	EventualDisruptionClass = "eventual" // eventual disruption is bounded by a NodePool's TerminationGracePeriod, regardless of blocking pod PDBs and the do-not-disrupt annotation
```

Comment:
I added it to `pkg/utils/disruption/disruption.go` instead. Adding it into `disruption/types.go` creates a circular import between the disruption and state packages since we need to use this constant in `statenode.go`.